### DISCUSSION TITLE
自定义菜单的使用方法

### DISCUSSION THREAD

**Discussion by hjwucc** [tangly1024/NotionNext#2195] [Q&A] [answered]
**描述bug**
除了已有菜单，如标签外，其他的自定义顶部菜单无法加载
**复现步骤**
更新最新版本>4.3.0后，出现了这个问题
**期望的正常结果**
正确加载自定义菜单
**截图**
![image](https://github.com/tangly1024/NotionNext/assets/44704392/da0a7168-2ae2-4e65-9d0d-f5a33e35ae80)
![image](https://github.com/tangly1024/NotionNext/assets/44704392/ac8ab81a-0c45-4207-b6ca-fa97c2e63ce9)

**环境**
- 【必填】NotionNext版本 [4.3.2]
- 【必填】主题 [nobelium]
- 【必填】部署方案 [vercel]
- 【可选】操作系统: [windows]
- 【可选】浏览器 [chrome]

**补充说明**
无

**Comment by tangly1024** [comment]
notion中添加Menu类型数据，将仅作为跳转功能使用，menu的slug字段将指定跳转的路径，而menu这条数据的**页面内容**没有任何实际作用。

menu的五种使用示例:

1. 添加menu ，title是博客 ， 状态是published , slug 是 https://blog.tangly1024.com ，则点击此菜单会跳转到 https://blog.tangly1024.com 这个地址

2. 添加menu， title是 关于， 状态是 published， slug 是 /about ，则点击此菜单会跳转到 您站点的/about 地址，如果您站点的域名是 https://test.com ，那么点击菜单会跳转到 https://test.com/about 路径 。此时您需要创建一篇slug为about、type是Page的、status是Published的文章，否则 打开https://test.com/about  会提示404.

3. 添加menu， title是 文章， 状态是 published， slug 是 /article/example-1 ，则点击此菜单会跳转到 您站点的 /article/example-1 地址，如果您站点的域名是 https://test.com ，那么点击菜单会跳转到 https://test.com/article/example-1 路径 。 此时您需要检查是否存在一篇 slug 为 example-1 ， type是Post、 Status是Published的文章，否则打开  https://test.com/article/example-1 会提示404

4. 添加menu， title是 分类-1， 状态是 published， slug 是 /category/test ，则点击此菜单会跳转到 您站点的 /category/test 地址，如果您站点的域名是 https://test.com ，那么点击菜单会跳转到 https://test.com/category/test 路径 。 此页面将聚合显示所有分类为test的文章 。

5. 添加menu， title是 标签-1， 状态是 published， slug 是 /tag/test ，则点击此菜单会跳转到 您站点的 /tag/test 地址，如果您站点的域名是 https://test.com ，那么点击菜单会跳转到 https://test.com/tag/test 路径 。 此页面将聚合显示所有打了test标签的文章 。


5. 添加menu， title是 归档， 状态是 published， slug 是 /archive，则点击此菜单会跳转到 您站点的  /archive 地址，如果您站点的域名是 https://test.com ，那么点击菜单会跳转到 https://test.com /archive 路径 。 此页面将聚合显示所有文章，并且按发布日期分组

**Comment by tangly1024** [comment, marked as answer]
相关菜单使用说明：
https://docs.tangly1024.com/article/notion-next-secondary-menu
